BangazonTests/ProductTypesTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+namespace BangazonTests
+{
+    // Author: William Kimball
+    // Description: Tests whether a product types page returns the correct content
+    public class ProductDetailTest : IClassFixture<WebApplicationFactory<Bangazon.Startup>>

```

Comment:
There is already a class in the BangazonTests namespace with the class name of ProductDetailsTest. Either rename the class and it's references or add your route to Elliot's test if applicable.